.github/workflows/data-loader-ci.yaml

```diff
@@ -0,0 +1,45 @@
+name: CI for CLI
+
+on:
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  data_loader_ci:
+    name: Build Data Loader CLI
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set version
```

Comment:
I might be missing something, but why is this needed?

Also, there are some differences between the original `ci.yaml`'s `check` job and this new workflow. Can you update this workflow based on the original `ci.yaml`? I guess it might fix the weird write permission issue.